modular_skyrat/modules/expeditionary-corps/code/jobs.dm

```diff
@@ -0,0 +1,299 @@
+/datum/job/expedition_commander
+	title = "Expedition Сommander"
+	description = ""
+	department_head = list(JOB_QUARTERMASTER)
+	faction = FACTION_STATION
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the quartermaster"
+	selection_color = "#d7b088"
+	//minimal_player_age = 40
+	exp_requirements = 400
+	exp_required_type = EXP_TYPE_CREW
+	exp_required_type_department = EXP_TYPE_SUPPLY
+	exp_granted_type = EXP_TYPE_CREW
+
+	departments_list = list(
+		/datum/job_department/cargo,
+		/datum/job_department/command,
+	)
+
+	outfit = /datum/outfit/job/expedition_commander
+	plasmaman_outfit = /datum/outfit/plasmaman/mining
+
+	paycheck = PAYCHECK_COMMAND
+	paycheck_department = ACCOUNT_CAR
+
+	display_order = JOB_DISPLAY_ORDER_EXP_CORPS
+	bounty_types = CIV_JOB_RANDOM
+
+	family_heirlooms = list(/obj/item/binoculars)
+
+	veteran_only = FALSE
```

Comment:
```suggestion
```
Redundant, it's false by default

Reply:
Likewise, I left it just in case, so that I could conveniently rewrite it later if necessary. I'll delete it.